apps/backoffice-v2/src/pages/Entity/components/CallToAction/CallToAction.tsx

```diff
@@ -30,7 +31,16 @@ const motionProps: ComponentProps<typeof MotionButton> = {
   animate: { y: 0, opacity: 1, transition: { duration: 0.2 } },
 };
 
-export const CallToAction: FunctionComponent<ICallToActionProps> = ({ value, data }) => {
+export const CallToAction: FunctionComponent<ICallToActionProps> = ({
+  value,
+  data,
+  documentSelection,
+  contextUpdateMethod,
+  revisionReasons,
+  rejectionReasons,
+  onReuploadReset,
+  onDialogClose,
```

Comment:
Please nest these under a property named `value.props` to match the other cells. `data` can be removed in favor of `props.id`, `props.disabled`, and `props.decision`, right now `value` is a string so it will be an object with a `text` property: `{ type: 'callToAction', value: { text: 'Reject', props: { id: 1, onDialogClose: () => ..., ... } }`